src/content/peerconnection/pr-answer/js/main.js

```diff
@@ -156,7 +156,7 @@ function accept() {
     onCreateAnswerError
   );
   btn2.disabled = true;
-  btn1.disabled = false;
+  btn1.disabled = true;
```

Comment:
Great, but could you update the button names to what they do (e.g call etc)? btn2 and btn1 are horrible names,